### PR TITLE
Add company survey example

### PR DESCRIPTION
**Pull request by Chilliwiddit**
## Purpose
Add company survey example

## Checklist
- [ ] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests
- [ ] Updated the spec
- [ ] Checked native-image compatibility

### CODE REVIEW

File: examples/companySurvey/ballerina.toml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+[package]
```
Comment (LakshanWeerasinghe) [inline examples/companySurvey/ballerina.toml:1]: Rename is file to `Ballerina.toml`

File: examples/companySurvey/main.bal [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import ballerina/io;
+import ballerinax/slack;
+
+public function companySurvey() returns error? {
+    slack:Client cl = check new ({
```
Comment (LakshanWeerasinghe) [inline examples/companySurvey/main.bal:5]: move this as a final module variable.

File: examples/companySurvey/main.bal [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+        auth: {
+            token: value
```
Comment (LakshanWeerasinghe) [inline examples/companySurvey/main.bal:7]: The token should be defined as configurable.

Comment (Chilliwiddit) [reply examples/companySurvey/main.bal]: Resolved Thanks

File: examples/companySurvey/main.bal (hunk 2 above)
Comment (Chilliwiddit) [reply examples/companySurvey/main.bal]: Resolved. Thank you.

File: examples/companySurvey/ballerina.toml (hunk 1 above)
Comment (Chilliwiddit) [reply examples/companySurvey/ballerina.toml:1]: Made the changes. Thanks

File: examples/companySurvey/Config.toml.template
Comment (NipunaRanasinghe) [inline examples/companySurvey/Config.toml.template:1]: No need to keep the `Config.toml` file

File: examples/companySurvey/ballerina.toml (hunk 1 above)
Comment (NipunaRanasinghe) [inline examples/companySurvey/ballerina.toml:1]:
```suggestion
```

File: examples/companySurvey/ballerina.toml [hunk 4]
```diff
@@ -0,0 +1,5 @@
...
+org = "wso2"
+name = "company_survey"
+version = "0.1.0"
+distribution = "2201.9.0"
```
Comment (NipunaRanasinghe) [inline examples/companySurvey/ballerina.toml:5]: missing newline

File: examples/companySurvey/main.bal [hunk 5]
```diff
@@ -0,0 +1,24 @@
+import ballerina/io; //importing the io module
```
Comment (NipunaRanasinghe) [inline examples/companySurvey/main.bal:1]: missing license header

File: examples/companySurvey/main.bal [hunk 6]
```diff
@@ -0,0 +1,24 @@
+import ballerina/io; //importing the io module 
+import ballerinax/slack; //importing the slack connector module
+
+configurable string value = ?; //initializing of the configurable token value variable
+
+final slack:Client slack = check new ({ //the initialization of the slack client at module variable
```
Comment (NipunaRanasinghe) [inline examples/companySurvey/main.bal:6]: No need to comment on self-explanatory code

File: examples/companySurvey/main.bal [hunk 7]
```diff
@@ -0,0 +1,24 @@
+import ballerina/io; //importing the io module 
+import ballerinax/slack; //importing the slack connector module
+
+configurable string value = ?; //initializing of the configurable token value variable
```
Comment (NipunaRanasinghe) [inline examples/companySurvey/main.bal:4]:
```suggestion
configurable string token = ?; 
```

File: examples/companySurvey/main.bal [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+
+final slack:Client slack = check new ({ //the initialization of the slack client at module variable
+    auth: {
+        token: value
```
Comment (NipunaRanasinghe) [inline examples/companySurvey/main.bal:8]:
```suggestion
        token
```

File: examples/companySurvey/main.bal [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+    }
+});
+
+public function companySurvey() returns error? {
```
Comment (NipunaRanasinghe) [inline examples/companySurvey/main.bal:12]:
```suggestion
public function main() returns error? {
```

File: examples/companySurvey/main.bal [hunk 10]
```diff
@@ -0,0 +1,24 @@
...
+
+    json conversationsResponse = check slack->/conversations\.create.post({name: "survey-coordination"});
+    io:println(conversationsResponse); //creation of a channel to coordinate surveys
```
Comment (NipunaRanasinghe) [inline examples/companySurvey/main.bal:15]: no need to print these intermediate reponses

**Review by NipunaRanasinghe — CHANGES_REQUESTED** [review]

**Review by NipunaRanasinghe — APPROVED** [review]

**Pull request merged**